src/components/Communication/CommunicationOptions/Error.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { ILayout, Layout } from "../Layout";
+import { TeamsTheme } from "../../../lib/withTheme";
+
+const DEFAULT_STRINGS = {
```

Comment:
Are we so confident that we know what strings users will want to use that we need defaults for these examples?

Reply:
If we will remove it, it means: those templates will be just a part of Storybook, users won't be able to import it from the HVC library.

From my understanding of the idea, this component not just a layout for messages which helps you to communicate through UI, but some "ready to use" cases.

I think, as a High-value component, it should have default values. Those strings are abstract and could cover most of the common cases.

Reply:
The benefit of it, that some of the string will be localized. And as other part of the library, those messages will speak to users with the same language.

@codepretty Let me know, what do you think.
It's easy to remove it from component.

Reply:
As we discussed offline, we will keep the defaults for now